frontend/kesaseteli/employer/public/locales/fi/common.json

```diff
@@ -11,6 +11,14 @@
     "retry": "Lataa sivu uudelleen",
     "logout": "Kirjaudu ulos"
   },
+  "loginPage": {
+    "logoutLabel": "Olet kirjautunut ulos",
```

Comment:
you could put `SWE` in the end of the sentence so it's easier to see that the translation works.